### PR TITLE
Update threat-hunting-in-threat-explorer.md

### PR DESCRIPTION
**Pull request by ajaj-shaikh**
Explorer V3 updates

### CODE REVIEW

**Comment by ajaj-shaikh** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]


I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term "You" includes me and my employer.
Contributor License Agreement
Contribution License Agreement

This Contribution License Agreement ("Agreement") is agreed to by the party signing below ("You"),
and conveys certain license rights to Microsoft Corporation and its affiliates ("Microsoft") for Your
contributions to Microsoft open source projects. This Agreement is effective as of the latest signature
date below.

  1.  Definitions.
"Code" means the computer software code, whether in human-readable or machine-executable form,
that is delivered by You to Microsoft under this Agreement.
"Project" means any of the projects owned or managed by Microsoft and offered under a license
approved by the Open Source Initiative (www.opensource.org<https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.opensource.org%2F&data=05%7C01%7Cajajshaikh%40microsoft.com%7C275d814c1eed4577c83508dacdfaabde%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638048773795312000%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=R9Cu4jmurCbhM0hBpADcdfnHvI1LqQyOeITD6lZqJrE%3D&reserved=0>).
"Submit" is the act of uploading, submitting, transmitting, or distributing code or other content to any
Project, including but not limited to communication on electronic mailing lists, source code control
systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
discussing and improving that Project, but excluding communication that is conspicuously marked or
otherwise designated in writing by You as "Not a Submission."
"Submission" means the Code and any other copyrightable material Submitted by You, including any
associated comments and documentation.
  2.  Your Submission. You must agree to the terms of this Agreement before making a Submission to any
Project. This Agreement covers any and all Submissions that You, now or in the future (except as
described in Section 4 below), Submit to any Project.
  3.  Originality of Work. You represent that each of Your Submissions is entirely Your original work.
Should You wish to Submit materials that are not Your original work, You may Submit them separately
to the Project if You (a) retain all copyright and license information that was in the materials as You
received them, (b) in the description accompanying Your Submission, include the phrase "Submission
containing materials of a third party:" followed by the names of the third party and any licenses or other
restrictions of which You are aware, and (c) follow any other instructions in the Project's written
guidelines concerning Submissions.
  4.  Your Employer. References to "employer" in this Agreement include Your employer or anyone else
for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
Submission is made in the course of Your work for an employer or Your employer has intellectual
property rights in Your Submission by contract or applicable law, You must secure permission from Your
employer to make the Submission before signing this Agreement. In that case, the term "You" in this
Agreement will refer to You and the employer collectively. If You change employers in the future and
desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
and secure permission from the new employer before Submitting those Submissions.
  5.  Licenses.

  *   Copyright License. You grant Microsoft, and those who receive the Submission directly or
indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
parties.
  *   Patent License. You grant Microsoft, and those who receive the Submission directly or
indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
Your patent claims that are necessarily infringed by the Submission or the combination of the
Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
import or otherwise dispose of the Submission alone or with the Project.
  *   Other Rights Reserved. Each party reserves all rights not expressly granted in this Agreement.
No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
granted by implication, exhaustion, estoppel or otherwise.

  1.  Representations and Warranties. You represent that You are legally entitled to grant the above
licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
have disclosed under Section 3). You represent that You have secured permission from Your employer to
make the Submission in cases where Your Submission is made in the course of Your work for Your
employer or Your employer has intellectual property rights in Your Submission by contract or applicable
law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
  2.  Notice to Microsoft. You agree to notify Microsoft in writing of any facts or circumstances of which
You later become aware that would make Your representations in this Agreement inaccurate in any
respect.
  3.  Information about Submissions. You agree that contributions to Projects and information about
contributions may be maintained indefinitely and disclosed publicly, including Your name and other
information that You submit with Your Submission.
  4.  Governing Law/Jurisdiction. This Agreement is governed by the laws of the State of Washington, and
the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
defenses of lack of personal jurisdiction and forum non-conveniens.
  5.  Entire Agreement/Assignment. This Agreement is the entire agreement between the parties, and
supersedes any and all prior agreements, understandings or communications, written or oral, between
the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft

Ajaj Shaikh

**Comment by ajaj-shaikh** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md [hunk 1]
```diff
@@ -64,11 +64,9 @@ In Microsoft Defender for Office 365, there are two subscription plans—Plan 1
 
 Defender for Office 365 Plan 1 uses *Real-time detections*, which is a subset of the *Threat Explorer* (also called *Explorer*) hunting tool in Plan 2. In this series of articles, most of the examples were created using the full Threat Explorer. Admins should test any steps in Real-time detections to see where they apply.
 
-After you go to **Explorer**, by default, you'll arrive on the **Malware** page, but use the **View** drop down to get familiar with your options. If you're hunting Phish, or digging into a threat campaign, choose those views.
+After you go to **Explorer**, by default, you'll arrive on the **All email** page, but use the **tabs** to navigate to the available views. If you're hunting Phish, or digging into a threat campaign, choose those views.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md]:
```suggestion
After you go to **Explorer**, by default, you'll arrive on the **All email** page, but use the tabs to navigate to the available views. If you're hunting phish or digging into a threat campaign, choose those views.
```

File: microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md [hunk 2]
```diff
@@ -64,11 +64,9 @@ In Microsoft Defender for Office 365, there are two subscription plans—Plan 1
...
 
-:::image type="content" source="../../media/view-drop-down.png" alt-text="The View dropdown in Threat Explorer" lightbox="../../media/view-drop-down.png":::
-
-Once a security operations (Sec Ops) person selects the data they want to see, whether the scope is narrow view like user **Submissions**, or a wider view, like **All email**, they can use the **Sender** button to further filter. Remember to select Refresh to complete your filtering actions.
+Once a security operations (Sec Ops) person selects the data they want to see, they can further narrow down the data by applying filters like Sender, Recipient, Subject etc. or select appropriate date range to get the desired results. Remember to select Refresh to complete your filtering actions.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md]:
```suggestion
Once a security operations (Sec Ops) person selects the data they want to see, they can further narrow down the data by applying filters such as Sender, Recipient, and Subject,  or select an appropriate date range to get the desired results. Remember to select Refresh to complete your filtering actions.
```

File: microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md [hunk 3]
```diff
@@ -77,13 +75,12 @@ Refining focus in Explorer or Real-time detection can be thought of in layers. T
...
-
-Clicking the **Advanced filter** pops a panel that will let Sec Ops hunters build queries themselves, letting them include or exclude the information they need to see. Both the chart and table on the Explorer page will reflect their results.
-
-:::image type="content" source="../../media/threat-explorer-chart-table.png" alt-text="The Results from a query" lightbox="../../media/threat-explorer-chart-table.png":::
+With the new version of Threat Explorer, users can use new dropdown options with four new operators on the filters-
+  - Equals any of – returns values matching the exact user input
+  - Equals none of - returns values not matching the exact user input
+  - Contains any of - returns values partially matching user input
+  - Contains none of - returns values not partially matching user input
+Note: The above filter conditions are availble based on filter types and input types.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md]:
```suggestion
With the new version of Threat Explorer, users can use the following new dropdown options with four new operators on the filters:
  - Equals any of – returns values matching the exact user input.
  - Equals none of – returns values not matching the exact user input.
  - Contains any of – returns values partially matching user input.
  - Contains none of – returns values not partially matching user input.

Note that these filter conditions are available based on filter types and input types.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ajaj-shaikh, kindly commit the suggested changes. Many thanks.

**Comment by ajaj-shaikh** [comment]
@MSFTTracyP can you please merge these changes?

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**